classes/class-wc-connect-taxjar-integration.php

```diff
@@ -398,10 +399,31 @@ public function _log( $message ) {
 	}
 
 	/**
-	 * @param $message
+	 * Display notice in cart or checkout page.
+	 *
+	 * @param string|array $message Error message.
+	 */
+	public function _notice( $message ) {
+		$formatted_message = is_scalar( $message ) ? $message : wp_json_encode( $message );
```

Comment:
Do we want to accept scalar values for the notice message? Or should we only be accepting strings?